Latex/Insight.bib

```diff
@@ -3279,7 +3279,7 @@ @Book{Reshetnyak1993
 
 @PhdThesis{ReveillesThese,
   Author         = "J-P. Reveilles",
-  Title          = "Geometrie discrete, calculs en nombres entiers et
+  Title          = "Geometrie discrete, calcul en nombres entiers et
```

Comment:
G{\\'e}om{\\'e}trie discr{\\`e}te, calcul (...)

Reply:
I explicitly didn't touch accents etc. as they are quite a bit harder to detect (as the might be in non-English titles, annotations etc as well as in the names of authors, editors etc.
I also didn't touch foreign languages parts, except the one mentioned, as the tool used didn't mention them or mentioned them (e.g. for `Geometrie` it suggested `Geometry`), due to the misspelling here I also noted the wrong word `calculs`. I see there are some more spelling mistakes here, corrected them as well.

Reply:
I understand; I simply suggested the change since the lines were already being looked at and modified.